daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md

```diff
@@ -189,22 +193,32 @@ It is recommended that a production-ready deployment includes the following sett
 
 6. Dapr also supports **scoping components for certain applications**. This is not a required practice, and can be enabled according to your security needs. See [here]({{< ref "component-scopes.md" >}}) for more info.
 
-
 ## Tracing and metrics configuration
 
 Dapr has tracing and metrics enabled by default. It is *recommended* that you set up distributed tracing and metrics for your applications and the Dapr control plane in production.
 
 If you already have your own observability set-up, you can disable tracing and metrics for Dapr.
 
 ### Tracing
+
 To configure a tracing backend for Dapr visit [this]({{< ref "setup-tracing.md" >}}) link.
 
 ### Metrics
+
 For metrics, Dapr exposes a Prometheus endpoint listening on port 9090 which can be scraped by Prometheus.
 
 To setup Prometheus, Grafana and other monitoring tools with Dapr, visit [this]({{< ref "monitoring" >}}) link.
 
+## Injector watchdog
+
+The Dapr operator service includes an _injector watchdog_ which can be used to detect and remediate situations where your app's pods may be deployed without the Dapr sidecar (i.e. the `daprd` container) when they should have been. For example, it can assist with recovering after a total cluster failure.
```

Comment:
```suggestion
The Dapr Operator service includes an _injector watchdog_ which can be used to detect and remediate situations where your application's pods may be deployed without the Dapr sidecar (the `daprd` container) when they should have been. For example, it can assist with recovering the applications after a total cluster failure.
```